tic-tac-toe-app/game.js

```diff
@@ -0,0 +1,83 @@
+        this.posTracker = {}
+        this.turn = 1
+        this.winner = null
+    }
+    
+    checkBoard(candidate){
+
+        for(let position in this.board){
+            this.posTracker[position] = this.board[position] 
+        }
+        //Checks each row to see if a player won
+        if(this.posTracker['0'] === this.posTracker['1'] && this.posTracker['0'] === this.posTracker['2'] && this.posTracker['0'] !== ''){
+            candidate % 2 === 0 ? game.playerOne.score += 1 : game.playerTwo.score += 1
+            this.gameOver() 
+        }
+        if(this.posTracker['3'] === this.posTracker['4'] && this.posTracker['3'] === this.posTracker['5'] && this.posTracker['3'] !== ''){
+            candidate % 2 === 0 ? game.playerOne.score += 1 : game.playerTwo.score += 1
+            this.gameOver()
+        }
+        if(this.posTracker['6'] === this.posTracker['7'] && this.posTracker['6'] === this.posTracker['8'] && this.posTracker['6'] !== ''){
+            candidate % 2 === 0 ? game.playerOne.score += 1 : game.playerTwo.score += 1
+            this.gameOver()
+        }
+        //Checks each diagonal row to see if a player won
+        if(this.posTracker['0'] === this.posTracker['4'] && this.posTracker['0'] === this.posTracker['8'] && this.posTracker['0'] !== ''){
+            candidate % 2 === 0 ? game.playerOne.score += 1 : game.playerTwo.score += 1
+            this.gameOver()
+        }
+        if(this.posTracker['2'] === this.posTracker['4'] && this.posTracker['2'] === this.posTracker['6'] && this.posTracker['2'] !== ''){
+            candidate % 2 === 0 ? game.playerOne.score += 1 : game.playerTwo.score += 1
+            this.gameOver()
+        }
+        //Checks each column to see if a player won
+        if(this.posTracker['0'] === this.posTracker['3'] && this.posTracker['0'] === this.posTracker['6'] && this.posTracker['0'] !== ''){
+            candidate % 2 === 0 ? game.playerOne.score += 1 : game.playerTwo.score += 1
+            this.gameOver()
+        }
+        if(this.posTracker['1'] === this.posTracker['4'] && this.posTracker['1'] === this.posTracker['7'] && this.posTracker['1'] !== ''){
+            candidate % 2 === 0 ? game.playerOne.score += 1 : game.playerTwo.score += 1
+            this.gameOver()
+        }
+        if(this.posTracker['2'] === this.posTracker['5'] && this.posTracker['2'] === this.posTracker['8'] && this.posTracker['2'] !== ''){
+            candidate % 2 === 0 ? game.playerOne.score += 1 : game.playerTwo.score += 1
+            this.gameOver()
+        }
+        //Checks to see if there is a draw
+        if(candidate >= 9){
+            console.log('Draw game')
+            alert('Draw Game')
+            location.reload()
+        } 
+        
+    }
+    
+    gameOver(){
+        this.board = ['', '', '', '', '', '', '', '', '']
+        if(game.turn % 2 !== 0){
+            alert('Player One Wins')
+            location.reload()
```

Comment:
The automatic reloading at the end of the game creates a solid user experience. 👍